src/propelauth/token_models.rs

```diff
@@ -23,8 +26,6 @@ pub struct User {
     pub username: Option<String>,
     #[serde(default)]
     pub properties: Option<HashMap<String, Value>>,
-    #[serde(default)]
-    pub metadata: HashMap<String, String>,
```

Comment:
Can't remove metadata for back-compat reasons

Reply:
Added back